src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --button-color: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408vh;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+img {
+  width: 160px;
+  height: 134px;
+  top: 33px;
+  left: 20px;
+}
```

Comment:
The 'img' selector is using the 'top' and 'left' properties without 'position' being set. These properties will not have any effect unless the position property is set to 'relative', 'absolute', 'fixed', or 'sticky'.